pkg/controller/controller_clusterservicebroker_test.go

```diff
@@ -286,6 +286,18 @@ func TestReconcileClusterServiceBrokerExistingServiceClassAndServicePlan(t *test
 	assertNumberOfActions(t, kubeActions, 0)
 }
 
+func TestReconcileClusterServiceBrokerSetOSBTimeOut(t *testing.T) {
```

Comment:
right now your test is failing because you have wrong expectation for `reconcileClusterServiceBroker` func

additionally, you can simplify assertion for expected timeout

Here is the working and simplified version of your test:
```go
func TestReconcileClusterServiceBrokerSetOSBTimeOut(t *testing.T) {
	const testOSBTimeOut = time.Second * 80

	_, _, _, testController, _ := newTestController(t, getTestCatalogConfig())
	testController.OSBAPITimeOut = testOSBTimeOut

	if err := reconcileClusterServiceBroker(t, testController, getTestClusterServiceBroker()); err != nil {
		t.Fatalf("Unexpected reconcileClusterServiceBroker failure: %v", err)
	}

	createdClient := testController.brokerClientManager.clients[NewClusterServiceBrokerKey(getTestClusterServiceBroker().Name)]
	if createdClient.clientConfig.TimeoutSeconds != int(testController.OSBAPITimeOut.Seconds()) {
		t.Errorf("Unexpected OSBTimeOut: got %v, expeted %v", createdClient.clientConfig.TimeoutSeconds, int(testController.OSBAPITimeOut.Seconds()))
	}
}
```

you can do sth similar to fix your test

Reply:
Oh, thanks for your nice review.